src/zenml/zen_stores/schemas/step_run_schemas.py

```diff
@@ -199,8 +199,6 @@ def to_model(
             workspace=self.workspace.to_model(),
             user=self.user.to_model() if self.user else None,
             parent_step_ids=parent_step_ids,
-            enable_cache=self.enable_cache,
-            enable_artifact_metadata=self.enable_artifact_metadata,
```

Comment:
I probably don't fully understand if these values are still needed or not, but to me this basically looks like you're ignoring these values even for step run entries that exist in the DB before your changes are applied.

Reply:
I actually didn't change anything for the step run schemas/models. These arguments were dead code before, the step run models don't even have these as fields 😅